src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `jv_jdbc_intro`.`books` (
+    `id` INT(10) NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255) NOT NULL,
+    `price` DECIMAL NOT NULL,
+    PRIMARY KEY (`id`)
+);
```

Comment:
each file should end with one empty line
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/49767049/bcf83ac1-19bf-468e-968f-eae0d382b02c)